beacon-chain/powchain/service.go

```diff
@@ -257,12 +257,13 @@ func (s *Service) Start() {
 			log.Fatal(err)
 		}
 		if genState == nil {
-			log.Fatal("cannot create genesis state: no eth1 http endpoint defined")
+			log.Warn("cannot create genesis state: no eth1 http endpoint defined")
```

Comment:
We do need eth1 http to create genesis state, partly the reason is we dont bake in genesis state like other clients

I believe downgrading the severity here will cause rest of the system to break

Have you tried this run time?

Reply:
I agree, this is useless before genesis, I tried this on pyrmont on a working dbase and it works fine, and on an empty dbase it just sits without doing anything as you would expect without an endpoint.

Reply:
Does your working db have a genesis state?

Reply:
yes, is my working pyrmont, on an empty dbase it just sits without doing anything.

Reply:
I don't think the issue is really about the genesis state or not genesis, but rather people that want to have their nodes without any connection to eth1. If my disk fills up and want to ditch geth and I don't want to run infura, this is what I will use. That's why I already had this patched when the request came in discord

Reply:
A wait, but I get your point without genesis this whole PR is ridiculous, will close with this comment. But I do not know if it's a good idea to bake in a genesis state though.